src/components/Announcement/index.tsx

```diff
@@ -0,0 +1,181 @@
+	[theme.breakpoints.up('sm')]: {
+		maxWidth: 72,
+	},
+	'& *': {
+		width: '100%',
+	},
+}));
+
+const AnnouncementTitle = styled('h3', {
+	name: 'Announcement',
+	slot: 'title',
+})(({ theme }) => ({
+	...theme.typography.bodyLg,
+	margin: 0,
+	display: 'flex',
+	alignItems: 'center',
+}));
+
+const AnnouncementContent = styled(Stack, {
+	name: 'Announcement',
+	slot: 'content',
+})(({ theme }) => ({
+	flexGrow: 1,
+	gap: theme.spacing(1),
+	[theme.breakpoints.up('sm')]: {
+		gap: theme.spacing(2),
+	},
+}));
+
+/**
+ * This component is a notification intended to be displayed in the bottom right
+ * corner of the screen, usually sharing updates on the product with our users.
+ *
+ * This is a Themed Component as documented here: https://mui.com/material-ui/customization/creating-themed-components/.
+ * As such, it can be modified in the theme.
+ */
+export const Announcement = React.forwardRef<HTMLDivElement, AnnouncementProps>(
+	function Announcement(inProps, ref) {
+		const props = useThemeProps({ props: inProps, name: 'Announcement' });
+		const {
+			illustration,
+			campaignId,
+			children,
+			title,
+			linkHref,
+			linkLabel,
+			onClose,
+			open,
+			sx,
+			...other
+		} = props;
+		const ownerState = {
+			...props,
+			campaignId,
+			children,
+			illustration,
+			title,
+			linkHref,
+			linkLabel,
+			onClose,
+			open,
+		};
+		const { state: analytics } = useAnalyticsContext();
+
+		// TODO remove this event after the first campaign, when we are sure that the notification is being shown properly
+		useEffect(() => {
+			analytics.webTracker?.track('Announcement: open', { campaignId });
+		}, []);
+
+		return (
+			<Snackbar
+				open={open}
+				anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+				TransitionComponent={(props: SlideProps) => {
+					return <Slide direction="up" {...props} />;
+				}}
+				ref={ref}
+				onClose={onClose}
+				sx={[
+					(theme) => ({ maxWidth: 350, zIndex: theme.zIndex.drawer - 1 }),
+					// See: https://mui.com/system/getting-started/the-sx-prop/#passing-the-sx-prop
+					...(Array.isArray(sx) ? sx : [sx]),
+				]}
+			>
+				<AnnouncementRoot ownerState={ownerState} {...other}>
+					{illustration && (
+						<AnnouncementIllustration>{illustration}</AnnouncementIllustration>
+					)}
+					<AnnouncementContent>
+						<AnnouncementTitle>
+							{title}
+							<IconButtonWithTracking
+								edge="end"
+								size="small"
+								eventName="Announcement: dismiss"
+								eventProperties={{ campaignId }}
+								onClick={(e) => {
+									if (onClose) {
+										onClose(e, 'closeButtonClick');
+									}
+								}}
+								sx={(theme) => ({ ml: 'auto', my: `-${theme.spacing(1)}` })}
+							>
+								<FontAwesomeIcon icon={faClose} />
+							</IconButtonWithTracking>
+						</AnnouncementTitle>
+						{children}
+						<MUILinkWithTracking
+							href={linkHref}
+							eventName="Announcement: click on link"
+							eventProperties={{ campaignId }}
```

Comment:
Perhaps we should track the link itself too? Maybe the link label too but that may be a stretch :P My thinking for the latter is maybe we can see what verbiage leads to more clicks